### DISCUSSION TITLE
In login page, make clear sponsors are from BTCPay Foundation

### DISCUSSION THREAD

**Discussion by ghost** [btcpayserver/btcpayserver#2269] [Ideas / Feature Requests]
In the login page, it currently displays the sponsors to the BTCPay Foundation as `Our Sponsors`.
![screenshot_20210209_083143](https://user-images.githubusercontent.com/30683012/107330053-59022300-6ab1-11eb-8770-c6024973c032.png)

It's of course nice to give credit to the vitally important financial sponsors to free software.

However, this message is on the user's server / website, that's NOT the BTCPay project / foundation website. Right now it looks like that the user gets sponsored by these companies.

### Suggestions

- Remove the sponsors entirely
- Make it obvious that these are BTCPay Foundation sponsors

**Comment by Kukks** [comment]
NACK to removing them, these are sponsors that allow this project to move forward by allowing contributors to not worry about their financial situation. If you want to remove them, you are free to customize the theme or fork. 

I'm ok with changing the copy to mention the foundation.

**Reply by ghost** [reply]
Yes, I think changing it to something like `Sponsors of the BTCPay Foundation` is the way to go.

**Reply by ghost** [reply]
I might misunderstand the code, but it seems that in current master, this text is already updated...?
https://github.com/btcpayserver/btcpayserver/blob/9e71c02eb9a67ec5dc35289409a28fd7abeda926/BTCPayServer/Views/Shared/_BTCPaySupporters.cshtml#L2

If yes, than this discussion is obsolete.

**Reply by dennisreimann** [reply]
Yes, I think so, Rockstar did that if I remember correctly.

**Reply by Eskyee** [reply]
> Yes, I think so, Rockstar did that if I remember correctly.

Looks good to me !!

**Comment by Eskyee** [comment]
I agree with kukks 💯 the foundation sponsors have to stay!!! It's what made the BTCPAY project possible... 👨🏽‍💻 future proof..
